docs/mobile-apps/automated-testing/espresso-xcuitest.md

```diff
@@ -1,25 +1,172 @@
 ---
 id: espresso-xcuitest
 title: Mobile App Testing with Espresso and XCUITest
-sidebar_label: Using Espresso and XCUITest
-hide_table_of_contents: true
+sidebar_label: Espresso and XCUITest
+description: Run Espresso and XCUITest projects on Sauce Labs.
 ---
 
-:::warning Espresso for VDC Now Available on saucectl
-**Espresso testing for virtual devices is now supported on `saucectl`, our CLI tool used with Testrunner Toolkit:**
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-* **[About saucectl](/testrunner-toolkit)**
-* **[saucectl Installation](/testrunner-toolkit/installation)**
-* **[saucectl CLI Reference](/testrunner-toolkit/saucectl)**
+Sauce Labs uses its framework agnostic test orchestrator [`saucectl`](/testrunner-toolkit) to execute Espresso and XCUITest tests based on one or more configuration files that instruct `saucectl` to run your tests exactly the way you specify. Results get published in your Sauce Labs dashboard, where you can compare 30 days of results across different environments and frameworks all in one view.
+
+## What You'll Need
+
+* Sauce Labs account `USERNAME` and `ACCESS_KEY` [Look them up](https://app.saucelabs.com/user-settings)
+* Your Espresso or XCUITest apps and files
+
+## Installation & Setup
+
+`saucectl` can execute both Espresso and XCUITest tests, so the set up steps are the same regardless of which framework you are using.
+
+### 1. Install `saucectl`
+
+Begin by installing the `saucectl` CLI in the root directory of your existing mobile application automated testing project or your intended directory.
+
+```bash
+curl -L https://saucelabs.github.io/saucectl/install | bash
+```
+
+:::caution Required Minimum Versions
+Espresso requires `saucectl` version 0.36.0 or later and XCUITest requires `saucectl` version 0.44.0 or later.
+:::
+
+### 2. Check out the demo repositories
+
+Clone or download the [Espresso](https://github.com/saucelabs/saucectl-espresso-example) and [XCUITest](https://github.com/saucelabs/saucectl-xcuitest-example) example repos, as applicable to your project, to obtain the `saucectl` directory structure and example files for use as templates.
+
+### 3. Link your Sauce Labs account
+
+1. Generate a credentials file that `saucectl` can reference to authenticate your CLI commands.      
+    ```
+    saucectl configure
+    ```
+1. Enter your Sauce Labs `USERNAME` and `ACCESS_KEY` at the prompts.
+1. Add the resulting `credentials.yml` file in the `.sauce` folder of your project directory to your `.gitignore` file to protect it from unintended exposure.
```

Comment:
Hmmm. Not directing user to move the file; just to add it to .gitignore so it won't be included in the upload to sauce. Maybe I should reword if it's not clear. I will play with this one. Good feedback.

Reply:
Let me rephrase to be sure it's clear.
Running `saucectl configure` will ___never___ create a `credentials.yml` file ___in the current directory___. 
User does not need to do anything except entering username/access key when it's prompted.

> 1. Add the resulting `credentials.yml` file in the `.sauce` folder of your project directory to your `.gitignore` file to protect it from unintended exposure.
=> This line can safely be totally removed.

Behind the scene, `saucectl` will actually create a `credentials.yml` file, but in a location where many programs stores their configuration (like `git` or `.vscode`), where it will not interfere with anything.
Since `.gitignore` is relative to the repository and `credentials.yml` is not within that said repository => There is no risk  of unintended exposure.